HIP/hip-873.md

```diff
@@ -0,0 +1,104 @@
+---
+hip: 873
+title: Adding Decimal Information to Token Balance Calls in the Hedera REST API
+author: Houdini NFT (@HoudiniNFT), Ty Smith (@ty-swirldslabs)
+requested-by: Atlantis
+type: Standards Track
+category: Mirror
+needs-council-approval: Yes
+status: Last Call
+last-call-date-time: 2024-02-14T07:00:00Z
+created: 2024-01-30
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/pull/873
+updated: 2024-01-31
+---
+
+### Abstract
+This HIP proposes an enhancement (with backwards compatibility) to the Hedera REST API to include decimal information in the token balance queries. Currently, users face challenges in accurately determining token balances due to the absence of decimal details in the `/tokens` and `/account/{id}/tokens` endpoints. This proposal aims to reduce the number of API calls needed and streamline the process of retrieving comprehensive token information.
```

Comment:
I guess we should also probably update `/api/v1/tokens/{tokenId}/balances` to include decimals as well.

Reply:
Added